src/Symfony/Component/Lock/Tests/Store/DoctrineDbalStoreTest.php

```diff
@@ -11,11 +11,16 @@
 
 namespace Symfony\Component\Lock\Tests\Store;
 
+use Doctrine\DBAL\Connection;
 use Doctrine\DBAL\DriverManager;
+use Doctrine\DBAL\Exception\TableNotFoundException;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
 use Symfony\Component\Lock\Key;
 use Symfony\Component\Lock\PersistingStoreInterface;
 use Symfony\Component\Lock\Store\DoctrineDbalStore;
 
+class_exists(\Doctrine\DBAL\Platforms\PostgreSqlPlatform::class);
```

Comment:
whats for?

Reply:
Check this comment: https://github.com/symfony/symfony/pull/44383#issuecomment-984200592